src/python/pants/engine/legacy/structs.py

```diff
@@ -8,6 +8,7 @@
 import os.path
 from abc import abstractproperty
 from builtins import object, str
+from collections import MutableSet, MutableSequence
```

Comment:
Use `pants.util.collections_abc_backport.py`, otherwise we get a deprecation warning on Py3.7